paper/paper.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'PyMilo: A Python Library for ML I/O'
+tags:
+  - Machine Learning
+  - Model Deployment
+  - Model Serialization
+  - Transparency
+  - MLOPS
+authors:
+  - name: AmirHosein Rostami
+    orcid: 0009-0000-0638-2263
+    corresponding: true
+    affiliation: 1
```

Comment:
If you would like to add your second affiliation, please take a look at [this link](https://joss.readthedocs.io/en/latest/paper.html#affiliations).

Reply:
@alirezazolanvari @sadrasabouri